flang/lib/Lower/OpenMP.cpp

```diff
@@ -159,7 +161,13 @@ static void createBodyOfOp(
     firOpBuilder.createBlock(&op.getRegion(), {}, tiv);
     for (auto &arg : args) {
       fir::ExtendedValue exval = op.getRegion().front().getArgument(argIndex);
-      [[maybe_unused]] bool success = converter.bindSymbol(*arg, exval);
+      auto val = fir::getBase(exval);
+      auto temp = firOpBuilder.createTemporary(
```

Comment:
I am not sure what is the requirement regarding the alloca, but it might be worth to add it in at least one of the lit test to ensure it is OK from an OpenMP perspective (i.e. "pinned" if needed).